src/clj/game/cards/events.clj

```diff
@@ -1199,6 +1214,12 @@
     :events {:pre-rez-cost nil}
     :end-turn {:effect (effect (unregister-events card))}}
 
+   "Spear Phishing"
+   {:implementation "Bypass is manual"
+    :prompt "Choose a server"
+    :choices (req runnable-servers)
+    :effect (effect (run target nil card))}
+   
```

Comment:
I decided against it due to the ways bypassing can be prevented, and I wasn't sure if it would break if new ice is added to the server during the run at a particular point.